populous/compat.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+from past.builtins import xrange
```

Comment:
I think I prefer the "forward-compatible" approach, so `from builtins import range`.

This means we need to import it everywhere `range` is used, and convert `range(...)` to `list(range(...))` where needed.